tests/future/api/conftest.py

```
@@ -0,0 +1,19 @@
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)